sql/core/src/test/scala/org/apache/spark/sql/MergeAsOfSuite.scala

```diff
@@ -194,4 +194,42 @@ class MergeAsOfSuite extends QueryTest with SharedSQLContext{
         Row(new Timestamp(48), "AAPL", 98.0, 100, null, null)
       ))
   }
+
+  test("generated tests") {
```

Comment:
This is not generated test. Please change the name.

Reply:
Add test chase that left key and right key are not strictly subsets of each other

Reply:
Add test caes:

* 17 / 17
* 6 / 17
* Cases where each key has more than one row
* Cases where left and right keys intersects